app/javascript/react/screens/App/Settings/__tests__/Settings.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Spinner } from 'patternfly-react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
```

Comment:
Let's set this up for all tests.  Can you please add

```
"snapshotSerializers": ["enzyme-to-json/serializer"]
```

to our [package.json](https://github.com/ManageIQ/manageiq-v2v/blob/master/package.json#L92)?

Then, we can just do

```js
expect(component).toMatchSnapshot();
```

Reply:
Sure! Didn't know you could do that.